packages/deployer/test/fixture-projects/custom-proxy/test/helpers/initializer.js

```diff
@@ -0,0 +1,28 @@
+const { TASK_DEPLOY } = require('@synthetixio/deployer/task-names');
+
+/**
+ * Initializer helper for the sample-project, allows to deploy and initialize
+ * the project on the given hardhat environment
+ * @param {import('hardhat/types').HardhatRuntimeEnvironment}
+ */
+module.exports = function createInitializer(hre) {
```

Comment:
I'm afraid you'll need to adapt this to the new way of initializing systems in tests that was just merged.
It's very easy tho. You can find some examples here: https://github.com/Synthetixio/synthetix-v3/pull/418/files